Automattic/wp-calypso
# JPC: provide extra section to Jetpack plans and features in plans pages.

This PR adds a section to the JPC plans pages with redirects to external plan/feature information.

This first iteration includes external redirects to jetpack.com. In the future if this proves helpful, we'd want to direct users to the  resources inside Calypso.

Added changes:

`http://calypso.localhost:3000/jetpack/connect/store`

![screen shot 2018-03-12 at 23 56 36](https://user-images.githubusercontent.com/13561163/37315261-b0930362-2650-11e8-9ceb-626121cf09ae.png)

## Update:

![screen shot 2018-03-27 at 20 49 46](https://user-images.githubusercontent.com/13561163/37991064-0be25370-3200-11e8-83d8-fcafaa1aabe0.png)


**To test:**
- checkout this branch in local Calypso,
-  go to JPC plans page `http://calypso.localhost:3000/jetpack/connect/store` and/or `http://calypso.localhost:3000/jetpack/connect/plans/:connected_site_slug` (where `connected_site_slug` is on a Free plan), and verify that the visual changes are visible and work well for all views,
- use `localStorage.debug = 'calypso:analytics:tracks'` to confirm that events associated to the buttons, `calypso_jpc_help_feature_comparison_click` & `calypso_jpc_help_plan_guide_click` , fire on click,
- ☝️ and redirect to correct sections at jetpack.com opening in a new tab
- please test on a variety of viewports
